sdk/src/test/java/io/customer/sdk/MockCustomerIOBuilder.kt

```diff
@@ -18,17 +18,20 @@ internal class MockCustomerIOBuilder {
         const val timeout = 6000
         val urlHandler = null
         const val shouldAutoRecordScreenViews = false
-    }
+        const val autoTrackDeviceAttributes = true
```

Comment:
As stated in a previous comment, there is a bug in the `CustomerIO.Builder.autoTrackDeviceAttributes`. Since `true` is the default value of `autoTrackDeviceAttributes` in the SDK, anytime that our automated tests run, it's not really testing the Builder. I suggest modifying the test function to test both true and false work or at least changing this to `false` to be opposite of SDK config's default value.

Reply:
Added test for changing values should reflect accordingly,